source/portable/mbedtls/core_pkcs11_mbedtls.c

```diff
@@ -4192,7 +4259,8 @@ CK_DECLARE_FUNCTION( CK_RV, C_SignInit )( CK_SESSION_HANDLE hSession,
     /* Check that a private key was retrieved. */
     if( xResult == CKR_OK )
     {
-        /* See explanation in prvCheckValidSessionAndModule for this exception. */
+        /* MISRA Ref 10.5.1 [Essential type casting] */
```

Comment:
Do we know why CK_BBOOL is not recognized as an unsigned char by coverity?

Reply:
Responded to in an IM.